wrapanapi/systems/container/rhopenshift.py

```diff
@@ -1233,7 +1277,28 @@ def set_meta_value(self, instance, key):
             'Provider {} does not implement get_meta_value'.format(type(self).__name__))
 
     def vm_status(self, vm_name):
-        raise NotImplementedError('vm_status not implemented.')
+        """Returns current vm/appliance state
+
+        Args:
+          vm_name: openshift project name
```

Comment:
Lets update this to be more specific, the openshift project name of the podified appliance.

Thoughts? Same for line 1294